sentences/nb/_common.yaml

```
@@ -92,52 +92,59 @@ lists:
         out: garage
       - in: <port>
         out: gate
-      - in: <persienne>
+      - in: <rullegardin>
         out: shade
       - in: <skodde>
         out: shutter
       - in: <vindu>
         out: window
+  lock_states:
+    values:
+      - in: låst
+        out: locked
+      - in: ulåst
+        out: unlocked
 expansion_rules:
   navn: "{name}[(en|et|n[s])]"
   område: "{area}[(n[s]|en|et[s]|met[s])]"
 
   #determinatives
   alle: "(alle|alt|samtlige)"
   i_på: "(i|på|ved [siden av])"
-  hva: "(hva er|hva er det|hvilken|hvilke)"
+  hva: "(hva[ er][ det]|hvilke[n])"
```

Shouldn't it be like this? If not, it can be:
* hva er det
* hva er
* hva det

And the last one is wrong.
```suggestion
  hva: "(hva[ er [det]]|hvilke[n])"
```